src/components/NewLocationPage/LocationName/utils.ts

```diff
@@ -43,7 +45,12 @@ export function getSplitRegionName(region: Region, condensed?: boolean) {
 function splitRegionName(regionName: string) {
   const splitRegion = regionName.split(' ');
   if (
-    regionSuffixes.includes(splitRegion[splitRegion.length - 1].toLowerCase())
+    regionSuffixes.includes(
+      splitRegion[splitRegion.length - 1].toLowerCase(),
+    ) &&
+    !multiSuffixIndicator.includes(
+      splitRegion[splitRegion.length - 2].toLowerCase(),
+    )
```

Comment:
wanna put this whole condition into a variable so its clearer exactly what its doing?

`const isSingleWordSuffix = regionSuffixes.includes(splitRegion[splitRegion.length - 1].toLowerCase()) && !multiSuffixIndicator.includes(splitRegion[splitRegion.length - 2].toLowerCase())`